BSP/Inc/BSP_I2C.h

```diff
@@ -8,22 +8,12 @@
 #include "BSP.h"
 #include "stm32f4xx_hal.h"
 #include "stm32f4xx_hal_i2c.h"
+#include "stm32f4xx_hal_fmpi2c.h"
 #include "stm32f4xx_hal_rcc.h"
 #include "stm32f4xx_hal_rcc_ex.h"
 #include "stm32f4xx_hal_gpio.h"
 #include "stdbool.h"
```

Comment:
the headers here should be system agnostic to support L4 and F4; i'd include <stm32xx_hal.h> (see how this is done in Tests/Test/blinky.c)